docs/modules/plugins/pages/plugin-json.adoc

```diff
@@ -5,6 +5,56 @@ The plugin provides a set of actions for transformation and validation of JSON d
 :plugin-short-name: json
 include::partial$plugin-installation.adoc[]
 
+== JSON Path
+
+JSON path expressions always refer to a JSON structure in the same way as XPath expression are used in combination with an XML document. The "root member object" in JsonPath is always referred to as `$` regardless if it is an object or array.
+
+JSON path expressions can use the dot–notation
+
+[source]
+----
+$.store.book[0].title
+----
+
+or the bracket–notation
+
+[source]
+----
+$['store']['book'][0]['title']
+----
+
+=== JSON Path Functions
+
+==== `distinct()`
+
+Vividus provides ability to use additional function for filtering repeated values.
+
+* `distinct()` - Provides an array containing only unique items from the input array.
+
+[source,json]
+----
+{
+    "type": [
+      "b",
+      "b",
+      "a",
+      "c",
+      "a"
+    ],
+    "name": [
+      "abc"
+    ]
+}
+----
+
+.distinct() function of JSON Path example
+|===
+|JSON Path |Result
+
+|`$.type.distinct()`
+|`["b","a","c"]`
```

Comment:
please provide more examples:
- array of objects
- what will happen if it's applied to object

Reply:
added example with array of objects
and for object - if I right understand, Jayway implementation by default remove repeating pairs name-value.
Example:
<img width="952" alt="Screenshot 2023-12-05 at 16 36 03" src="https://github.com/vividus-framework/vividus/assets/35694955/866764a1-be14-4a47-bd2e-517689952aaf">